### ISSUE TITLE
[Chromaticraft] Blitz spawning in Chroma Dimension

### ISSUE THREAD

**Issue by tiny-maple** [ReikaKalseki/Reika_Mods_Issues#2438]
![2020-04-19_14 03 58](https://user-images.githubusercontent.com/62176592/79680875-40393000-8247-11ea-9f57-de37e37762af.png)
I discovered that only Blitz are spawning in structure field. I have also visited other structure field and found blitz too.
DragonAPI v25c
Chromaticraft v25b
Playing in singleplayer survival

**Comment by ReikaKalseki** [comment]
The hell? The spawn list is empty and I have explicit disallow of hostile spawns. By what code path logic do these spawn normally?

**Comment by tiny-maple** [comment]
![2020-04-22_01 57 39](https://user-images.githubusercontent.com/62176592/79977272-940a7a00-84d0-11ea-80c8-2ddcf8a408cd.png)
Feels like this is related, i flew around chroma dimension and saw that biome is plains instead of "structure field".

**Comment by ReikaKalseki** [comment]
That is definitely related, and indicative of some sort of map corruption or ID issues.

**Comment by tiny-maple** [comment]
Odd, because Plains are id 1 and structure id is 107 by default, and that is intended to prevent conflict of biome ID. Seems like the world did not get reset properly before a new  generation.
![2020-04-22_13 16 59](https://user-images.githubusercontent.com/62176592/80089000-dcd53800-858f-11ea-967a-fa96f9ae0654.png)
I get this kind of generation frequently and even more so in the far lands.

**Comment by ReikaKalseki** [comment]
That is indicative of chunks not being deleted.

**Comment by tiny-maple** [comment]
![2020-04-26_20 31 31](https://user-images.githubusercontent.com/62176592/80308430-9ab32d00-8801-11ea-8da1-47a4b1246db0.png)
I'm not sure what cause it, but I manage to reproduce it. 
![2020-04-26_20 32 53](https://user-images.githubusercontent.com/62176592/80308462-cafacb80-8801-11ea-94c6-a18f59b43844.png)
If you race into unloaded chunks and while having cause the game to freeze up (something causing the game memory use to reach 90 to 100% in chroma dimension only) it would temporary cause the biome to change to plains. I'm not sure why does farland biome is able to revert back to "Sparking sand" but "structure field" is unable to and remains as plains, allowing blitz to spawn. 
Here some worldgen cause by this:
![2020-04-26_20 26 38](https://user-images.githubusercontent.com/62176592/80308560-4bb9c780-8802-11ea-8a18-41cb6b193999.png)
![2020-04-26_20 24 35](https://user-images.githubusercontent.com/62176592/80308569-52e0d580-8802-11ea-8108-8460c89c5a99.png)

**Comment by OvermindDL1** [comment]
Two questions of who sees this issue, are you using Optifine and are you in SinglePlayer?  Optifine screws with the spawning maps in singleplayer, no I don't know why...

**Comment by tiny-maple** [comment]
>Optifine screws with the spawning maps in singleplayer, no I don't know why...

First screenshot was used without optifine, and blitz spawned, rest of the screenshot had blitz spawn and optifine was used. Both are played in singleplayer world. I don't think optifine is the cause.

**Comment by OvermindDL1** [comment]
Yeah if it happened without optifine then that wasn't it, thanks for the followup.  :-)
